packages/f-checkout/test/specs/component/f-checkout-delivery.component.spec.js

```diff
@@ -17,9 +17,22 @@ describe('f-checkout "delivery" component tests', () => {
         expect(CheckoutComponent.doesInputFieldExist(field)).toBe(true);
     });
 
-    forEach(['mobileNumber'])
-    .it('should display the mandatory fields', field => {
+    it('should display the mandatory fields', () => {
         // Assert
-        expect(CheckoutComponent.isFieldDisplayed(field)).toBe(true);
+        expect(CheckoutComponent.isFieldDisplayed('mobileNumber')).toBe(true);
+    });
+
+    it('should display an error message when a number format is incorrectly entered', () => {
```

Comment:
Ok, I still think this test needs the reverse tho because that particular test is looking for a correct submission rather than a the scenario around showing an error message, it doesn't look for the error message in the same way as this one does. This one is more granular.

Reply:
Ok, I've added that extra test now and also renamed the tests so its a little clearer.